### PR TITLE
Add PLX Keyboard

### PR DESCRIPTION
**Pull request by 2Moons-JP**
## Description

PLX Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [x ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plx/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+![PLX](https://i.imgur.com/xJvv1kq.jpg)
+
+An 60% gasket mounted board with a Tsangan layout and a 7 degree typing angle.
+
+* Keyboard Maintainer: [2Moons](https://github.com/2Moons-JP)  
+* Hardware Supported: PLX PCB, ATMEGA32U4-AU
+* Hardware Availability: [Website](https://peac.design)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/plx/readme.md:11]: Could you add instructions on how to enter the bootloader to flash new firmware?  Eg, is there a physical reset button? etc.

**Review by drashna — APPROVED** [review]

File: keyboards/plx/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# PLX
+
+![PLX](https://i.imgur.com/xJvv1kq.jpg)
```
Comment (fauxpark) [inline keyboards/plx/readme.md:3]:
```suggestion
![PLX](https://i.imgur.com/xJvv1kqh.jpg)
```

File: keyboards/plx/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+An 60% gasket mounted board with a Tsangan layout and a 7 degree typing angle.
+
+* Keyboard Maintainer: [2Moons](https://github.com/2Moons-JP)
```
Comment (fauxpark) [inline keyboards/plx/readme.md:7]:
```suggestion
* Keyboard Maintainer: [2Moons](https://github.com/2Moons-JP)
```

File: keyboards/plx/keymaps/default/rules.mk [hunk 4]
```diff
@@ -0,0 +1,2 @@
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no
```
Comment (fauxpark) [inline keyboards/plx/keymaps/default/rules.mk:2]: This file can be deleted.

File: keyboards/plx/rules.mk [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = no      # Audio control and System control
+CONSOLE_ENABLE = no      # Console for debug
+COMMAND_ENABLE = no      # Commands for debug and configuration
```
Comment (fauxpark) [inline keyboards/plx/rules.mk:14]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**